src/scripts/main.js

```diff
@@ -20,9 +20,52 @@ const food = {
 };
 
 const tree = document.querySelector('#tree');
+const [drink, fruit] = Object.keys(food);
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  tree.insertAdjacentHTML('afterbegin', `
```

Comment:
It works! But let's implement a solution using JS, not HTML. You should write a universal solution (what if there would be 100 elements in the list, with 5 levels of nesting - difficult to handle by hand)